src/main/scala/it/pagopa/interop/purposeprocess/api/impl/RiskAnalysisValidation.scala

```diff
@@ -52,12 +65,59 @@ object RiskAnalysisValidation {
     )
   }
 
+  private def validateFormWithRulesOnlyFormal(
```

Comment:
what if instead of duplicating these methods, we just add a `schemaOnlyValidation` flag, which let validations skip steps not required?
As far as I can see, the validations to skip are:
- `validateExpectedFields`
- `validateDependency` in the `validateField` method